hydra/_internal/defaults_list.py

```diff
@@ -390,13 +390,15 @@ def _update_overrides(
                     f"""\
                     In {parent.get_config_path()}: Invalid overriding of {d.group}:
                     Default list overrides requires 'override' keyword.
+                    Note: in Hydra 1.2, overrides will only be allowed to appear at the end of the defaults list.
```

Comment:
This information is appears in red in the migration link. I don't think we need it in the error message.